kit/signals/context.go

```diff
@@ -27,5 +26,5 @@ func WithSignals(ctx context.Context, sigs ...os.Signal) context.Context {
 
 // WithStandardSignals cancels the context on os.Interrupt, syscall.SIGTERM.
 func WithStandardSignals(ctx context.Context) context.Context {
-	return WithSignals(ctx, os.Interrupt, syscall.SIGTERM)
+	return WithSignals(ctx, os.Interrupt, os.Kill)
```

Comment:
This is a tangential cleanup/refactor, to use platform-independent `os.Kill` instead of platform-dependent `syscall.SIGTERM`